docs/tutorials/01_introductory/viz_solar_system.py

```diff
@@ -293,7 +293,7 @@ def timer_callback(_obj, _event):
 
         update_track(positions_neptune, neptune_track, neptune_orbit_actor)
 
-    if cnt == 1500:
+    if cnt == 3000:
```

Comment:
Can you put it back? The simulation is too long for the doc generation

Reply:
Should I change the equation back too?